src/layouts/Titled/components/TOC.tsx

```diff
@@ -121,6 +121,9 @@ export const TOC: React.FC<{ data: IHeading[] }> = ({ data }) => {
         position: sticky;
         top: ${p => p.theme.space.lg};
         margin-left: ${p => p.theme.space.base * 15}px;
+        max-height: ${OFFSET}px;
```

Comment:
This wants to be a `calc` of something like `(100vh - 64px)` - where 64 pixels is the existing top and bottom 32px of padding. Also, the width might need some design love in order to better accommodate the scrollbar size.

Reply:
Since the width of the TOC list isn't explicit, should `padding-right` be added to the TOC list to account for the scrollbar size? I think `4px` could work. Any suggestions come to mind? What do ya'll think? 

@allisonacs @rossmoody

Reply:
We have research and helpful guidance under a now-deprecated CSS component – https://garden.zendesk.com/css-components/scrollbars/ – to understand that standard "rail" width for a scrollbar is 15px. In this case, I think we can use our base-4 and apply an extra 16px of space.

Reply:
@hzhu I might do something more like `20px`. The default width of a scroll bar on macos is 7px, but it's 17 on windows.

Reply:
Awesome thanks for the size suggestions. I went with a larger, `16px` space.